numpy/core/multiarray.py

```diff
@@ -1107,6 +1107,21 @@ def copyto(dst, src, casting=None, where=None):
         A boolean array which is broadcasted to match the dimensions
         of `dst`, and selects elements to copy from `src` to `dst`
         wherever it contains the value True.
+
+    Examples
+    --------
+    >>> A = np.array([4,5,6])
+    >>> B = [1,2,3]
+    >>> np.copyto(A,B)
+        A
```

Comment:
You need to add the `>>>` to the beginning of each and every command lines, which tells the testing infrastructure which ones are the one to run.
Here and below:
```suggestion
    >>> np.copyto(A, B)
    >>> A
```